code-challenges.js

```diff
@@ -39,3 +95,11 @@ var numbersToAdd2 = [0, 7, -8, 12]
 // Expected output: [0, 7, -1, 11]
 var numbersToAdd3 = []
 // Expected output: []
+
+
+
+const cumulativeSum = (sum => value => sum += value)(0);
```

Comment:
this is really beautiful, concise code but you may want to add comments. it's not immediately clear what the code is doing.